### ISSUE TITLE
UX/UI: improve the CSS

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-lite#2894]
Nothing again you, @DamnClin :-D
The CSS can be improved

**Comment by DamnClin** [comment]
What???? This lovely handmade with heart shitty page can be improved? In fact it's more a must be than a can be (but I can't do it, totally impossible for me)...

**Comment by Gnuk** [comment]
I may be interested to contribute to this but with something like Tikui integration before. What do you think about it ?

**Comment by pascalgrimaud** [comment]
I don't have any objection to use Tikui, and it would be very good use case :)

But, what to keep in mind:
- the application JHLite should work locally, without the need of external server (nginx or something else)
- the application JHLite should work offline
- it means, the CSS should be directly included in the project

I don't know what would be better:
- create a new repository ? (I'd like to avoid if possible)
- include it directly in the project, but where in the structure ?

Plz, don't start something before we all agree, I don't wanna waste your time

**Comment by Gnuk** [comment]
The idea I have is to includes Tikui as a module and have the style in the project structure and serve the style like vue/angular/react are served. The only thing I need to achieve this is to contribute to Tikui to let use another directory for sources and generation (will be set by default to src and dust like today but could be overridden by a configuration to src/main/... )

**Comment by pascalgrimaud** [comment]
Link for reference:
- Design: https://github.com/jhipster/jhipster-lite/issues/1565
- Landscape: https://github.com/jhipster/jhipster-lite/pull/3022

**Comment by Gnuk** [comment]
Maybe you're interested about this issue @ascelineboullen

**Comment by pascalgrimaud** [comment]
Closing in favor of https://github.com/jhipster/jhipster-lite/issues/10576
